### DISCUSSION TITLE
After launching up the Karavan designer Getting the 404 page not found

### DISCUSSION THREAD

**Discussion by kondetiramanakumar** [apache/camel-karavan#1225] [Q&A] [answered]
<img width="854" alt="image" src="https://github.com/apache/camel-karavan/assets/14121796/3c062fc8-3026-415c-ab33-93728b673a2c">
please help me out from this.

**Comment by kondetiramanakumar** [comment]
@mgubaidullin please help me out from this errors while launching the karavan designer.

**Comment by mgubaidullin** [comment, marked as answer]
Karavan should be used as web application or VS Code extension.
Pure Designer is for Karavan development only.

To build Desinger, first generate Typescript models, Kamelets and Components. 
Then install karavan-core, after that Designer might be build.

The simplest way is to follow Github Actions that build application https://github.com/apache/camel-karavan/blob/4.4.0/.github/workflows/app.yml
